src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,56 @@
+package duke;
+
+import java.io.*;
+import java.util.ArrayList;
+
+public class Storage {
+  String path;
+
+  public Storage(String path) {
+    this.path = path;
+  }
+
+  public void saveBaby(ArrayList<Task> taskList) throws IOException {
+    BufferedWriter taskWriter = new BufferedWriter(new FileWriter(path));
+    StringBuilder tasks = new StringBuilder();
+    for (Task task : taskList) {
+      tasks.append(task.toSaveString()).append("\n");
+    }
+    taskWriter.write(tasks.toString());
+    taskWriter.close();
+  }
+
+  public void loadBaby(TaskList taskList, Parser parser) throws IOException {
+    File file = new File(path);
+    if (file.createNewFile()) return;
```

Comment:
Depending on the codestyle, you should place the return in Egyptian case, even though this way is neater